books/RayTracingInOneWeekend.html

```diff
@@ -2996,8 +3243,8 @@
 Defocus Blur
 ====================================================================================================
 
-Now our final feature: defocus blur. Note, all photographers will call it “depth of field” so be
-aware of only using “defocus blur” among friends.
+Now our final feature: defocus blur. (Photographers will call it “depth of field”, so just use the
+term “defocus blur” among your raytracing friends.)
```

Comment:
I actually kind of like the original sentence. With modification:

Now our final feature: _defocus blur_. Note, photographers will call it _depth of field_ so be
sure to only use _defocus blur_ among friends.

Reply:
Took this sentence, but kept “... among your raytracing friends.” I mean, some friends can be photographers. :)